src/metabase/task/sweep_query_analysis.clj

```diff
@@ -98,18 +98,14 @@
         job     (jobs/build
                  (jobs/of-type SweepQueryAnalysis)
                  (jobs/with-identity job-key)
-                 (jobs/store-durably)
-                 (jobs/request-recovery))
+                 (jobs/store-durably))
         trigger (triggers/build
                  (triggers/with-identity (triggers/key "metabase.task.backfill-query-fields.trigger"))
-                 (triggers/start-now)
```

Comment:
This kind of compound trigger does not seem to work properly, stick to `trigger-now!`, which creates another temporary trigger.